src/server/json_family.cc

```diff
@@ -70,6 +73,41 @@ string JsonType(const json& val) {
   return "";
 }
 
+template<class Json>
+error_code json_replace_with_ec(Json& instance, string_view& path, JsonReplaceCb callback,
```

Comment:
the jsoncons library uses templated classes for maximum flexibility. Here, I suspect you do not need it since we always use `json` type. Why not get rid of the template? Also, the naming convention for function names is `CamelCase`.